apps/src/templates/projects/ProjectCardGrid.jsx

```diff
@@ -23,90 +23,68 @@ const styles = {
+      playlab: PropTypes.arrayOf(projectPropType),
+      artist: PropTypes.arrayOf(projectPropType),
+    }).isRequired,
+    galleryType: PropTypes.oneOf(['personal', 'class', 'public']).isRequired
   },
 
   render() {
-    const { projects, galleryType } = this.props;
+    const { projectLists, galleryType } = this.props;
 
-    if (galleryType === 'public') {
-      return (
-        <div style={styles.grid}>
-          <h2 style={styles.labHeading}> {i18n.projectTypeApplab()} </h2>
-          {this.groupByType(projects).applab.slice(0,4).map((project, index) => (
-            <div key={index} style={styles.card}>
-              <ProjectCard
-                projectData={project.projectData}
-                currentGallery={galleryType}
-              />
-            </div>
-          ))}
-          <h2 style={styles.labHeading}> {i18n.projectTypeGamelab()} </h2>
-          {this.groupByType(projects).gamelab.slice(0,4).map((project, index) => (
-            <div key={index} style={styles.card}>
-              <ProjectCard
-                projectData={project.projectData}
-                currentGallery={galleryType}
-              />
-            </div>
-          ))}
-          <h2 style={styles.labHeading}> {i18n.projectTypeArtist()} </h2>
-          {this.groupByType(projects).artist.slice(0,4).map((project, index) => (
-            <div key={index} style={styles.card}>
-              <ProjectCard
-                projectData={project.projectData}
-                currentGallery={galleryType}
-              />
-            </div>
-          ))}
-          <h2 style={styles.labHeading}> {i18n.projectTypePlaylab()} </h2>
-          {this.groupByType(projects).playlab.slice(0,4).map((project, index) => (
-            <div key={index} style={styles.card}>
-              <ProjectCard
-                projectData={project.projectData}
-                currentGallery={galleryType}
-              />
-            </div>
-          ))}
-          <h2 style={styles.labHeading}> {i18n.projectTypeWeblab()} </h2>
-          {this.groupByType(projects).weblab.slice(0,4).map((project, index) => (
-            <div key={index} style={styles.card}>
-              <ProjectCard
-                projectData={project.projectData}
-                currentGallery={galleryType}
-              />
-            </div>
-          ))}
-        </div>
-      );
-    }
     return (
       <div style={styles.grid}>
-        {this.sortByDate(projects).map((project, index) => (
+        <h2 style={styles.labHeading}> {i18n.projectTypeApplab()} </h2>
+        {projectLists.applab && projectLists.applab.slice(0,4).map((project, index) => (
+          <div key={index} style={styles.card}>
+            <ProjectCard
+              projectData={project.projectData}
+              currentGallery={galleryType}
+            />
+          </div>
+        ))}
+        <h2 style={styles.labHeading}> {i18n.projectTypeGamelab()} </h2>
```

Comment:
Lot of repeated code across these. Wonder if we should just make a simple ProjectCardList class (maybe even in the same file). Could be as simple as:
```jsx
const ProjectCardList = ({list}) => ({
  list.map((project, index) => (
    <div ...>
      <ProjectCard .../>
   </div>
  )
})